indico/modules/rb_new/client/js/setup.jsx

```diff
@@ -20,6 +21,17 @@ export default function setup(overrides = {}, postReducers = []) {
         const appContainer = document.getElementById('rb-app-container');
         const store = createRBStore(overrides, postReducers);
 
+        let oldPath = history.location.pathname;
+        history.listen(({pathname: newPath}) => {
```

Comment:
Couldn't this be rewritten as a reducer (on `LOCATION_CHANGE`)?

Reply:
reducers are not supposed to dispatch new actions

Reply:
Ah right, I didn't see you have to fetch new stuff. If we saved it on the redux state when saving it remotely, then we could just use a reducer. But yeah, probably not worth the trouble.